Communication/Table/TokenPositionRequestHandler.gd

```diff
@@ -6,10 +6,13 @@ class_name TokenPositionRequestHandler
 # {
 #    "keyword": "SET_TOKEN_POSITION",
 #    "object_id": 0,
-#    "position": [123.0, 123.0]
+#    "position_x": 123.0,
+#    "position_y": 123.0
 # }
 #
 
+var table_communicator: TableCommunicator  # To be injected
```

Comment:
re. the first message: yeah, there are some other patterns we could try using for that. I don't think it's a problem in this case because these will only be used in that specific structure, but if we find something nice that works for everything, we can adapt this too.

re. the second message: I agree, that's basically what the "# FIXME: Maybe we should have a `add_attribute_mapping` instead and leave the object creation up to the user." would amount to: when the user can pass any class that derives from `GameObjectAttribute`, then that class is essentially a functor, which is similar to a callback (but a bit cleaner for something like this, in my opinion). So any weird kind of attribute calculation could be passed. So yes, more complex use cases like that are definitely supported!